packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -586,6 +586,7 @@ export abstract class BaseService extends Resource
 
   /**
    * This method is called to create a networkConfiguration.
+   * @deprecated use configureAwsVpcNetworkingWithMultipleSecurityGroups instead.
```

Comment:
update to use configureAwsVpcNetworkingWithSecurityGroups

Reply:
🤦‍♂️ Done!